config/routes.rb

```diff
@@ -8,7 +8,7 @@
 
   namespace :admin do
     resources :dashboard, only: [:index]
-    resources :items, only: [:index, :edit]
+    resources :items, only: [:index, :edit, :new, :create, :update]
```

Comment:
Looking good! It's making think of why/how we had an item  edit page but not the update action for it.